File: CursoCsharpUdemy/exer/udemy/EstruturaSequencial/Exer05.cs
using System;

namespace exer.udemy.EstruturaSequencial
{
    public class Exer05
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Informe o código da peça 1, o número de peças 1, o valor unitário de cada peça 1: ");
            string[] values = Console.ReadLine().Split(' ');
            int code1 = int.Parse(values[0]);
            int qty1 = int.Parse(values[1]);
            double price1 = double.Parse(values[2], CultureInfo.InvariantCulture);

            Console.WriteLine("Informe o código da peça 2, o número de peças 2, o valor unitário de cada peça 2: ");
            values = Console.ReadLine().Split(' ');
            int code2 = int.Parse(values[0]);
            int qty2 = int.Parse(values[1]);
            double price2 = double.Parse(values[2], CultureInfo.InvariantCulture);

            double total = price1 * qty1 + price2 * qty2;

            Console.WriteLine("Total a pagar: R$ " + total.ToString("F2", CultureInfo.InvariantCulture));
        }
    }
}

File: CursoJava/old/Curso de Java/Repeticao-Sequencial-Condicional/src/Exer/EstruturaSequencial/Exer05.java
package Exer.EstruturaSequencial;

import java.util.Scanner;

public class Exer05 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Digite o código da peça, seu número, o valor un da peça 1: ");
	    int cod1 = sc.nextInt();
	    int qty1 = sc.nextInt();
	    double price1 = sc.nextDouble();
	    
	    System.out.print("Digite o código da peça, seu número, o valor un da peça 2: ");
	    int cod2 = sc.nextInt();
	    int qty2 = sc.nextInt();
	    double price2 = sc.nextDouble();

	    double total = price1 * qty1 + price2 * qty2;

	    System.out.printf("VALOR A PAGAR: R$ %.2f%n", total);
	    System.out.println("Peças compradas: ");
	    System.out.println("ID peça: " + cod1 + " // " + "Quantidade " + qty1);
	    System.out.println("ID peça: " + cod2 + " // " + "Quantidade " + qty2);
	    
		sc.close();
	}

}
